### PR TITLE
[11.1.X][Jemalloc] Track the peak memory used by each thread

### PR DESCRIPTION
**Pull request by smuzaffar**
follow up of #5853 where we have mistakenly used the wrong tag to build

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

File: jemalloc.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### RPM external jemalloc 5.2.1
 
-%define tag %{realversion}
+%define tag 8b5961b8a2ce67f1182f13ed5ce1adad3e2e99c1
```
Comment (fwyzard) [inline jemalloc.spec]:
```suggestion
%define tag de1caefb587217f0b519eb425d7a9b3570e5ba28
```

Comment (fwyzard) [reply jemalloc.spec]: @smuzaffar could you pick the latest updated to jemalloc ?

**Comment by fwyzard** [comment]
please test

**Comment by silviodonato** [comment]
merge

**Pull request merged**